spec/services/proceeding_type_full_text_search_spec.rb

```diff
@@ -52,7 +52,7 @@
 
       it "returns twelve records" do
         result_set = proceeding_type_full_text_search_results
-        expect(result_set.map(&:ccms_code).sort).to eq %w[SE013 SE013A SE013E SE014 SE014A SE014E SE015 SE015A SE015E SE016 SE016A SE016E]
+        expect(result_set.map(&:ccms_code).sort).to eq %w[PB029 PB030 SE013 SE013A SE013E SE014 SE014A SE014E SE015 SE015A SE015E SE016 SE016A SE016E]
```

Comment:
@naseberry This is testing that the `additional_search_values` are working correctly. I've updated this now though (in the latest commit) to remove some of these values. They're not needed because 'cao' is in the `meaning` now. And I've updated the spec too